README.md

```diff
@@ -136,6 +136,8 @@ Contributions are welcome in the form of GitHub pull requests. However, the `pan
 
 Instead, to propose a change, please fork [pantheon-systems/wordpress-composer-managed](https://github.com/pantheon-systems/wordpress-composer-managed) and submit a PR to that repository.
 
+Feature or bugfix development branches/pull requests should originate from and be submitted against the `default` branch. Please note that the `main` branch is the main _test_ branch and exists as a clone of the default branch for use in testing automation. 
```

Comment:
I'm unclear why the `main` branch exists if it is always a copy of the default branch. Also, the comment above seems to be for the pantheon-systems/wordpress-composer-managed repo, whereas the comments above it appear to be for the pantheon-upstream/wordpress-composer-managed repo, and it's not clear to me which is out of place (or whether, perhaps, I am simply confused about the purpose of this change).

Reply:
It seems that you should have two README files, one that is displayed in this project, and one that is copied into pantheon-upstream/wordpress-composer-managed. This is already done for the decoupled upstream, but not for pantheon-upstream/wordpress-composer-managed.

Reply:
Updated